pkg/image/prune/prune.go

```diff
@@ -244,10 +244,16 @@ func (*dryRunRegistryPinger) ping(registry string) error {
 // Also automatically remove any image layer that is no longer referenced by any
 // images.
 func NewPruner(options PrunerOptions) Pruner {
-	g := graph.New()
-
-	glog.V(1).Infof("Creating image pruner with keepYoungerThan=%v, keepTagRevisions=%v, pruneOverSizeLimit=%v",
-		options.KeepYoungerThan, options.KeepTagRevisions, options.PruneOverSizeLimit)
+	keepTagRevisions := "<nil>"
```

Comment:
`"<nil>"` reveals an implementation detail. To a regular user, it may look like a magic. Would it be perhaps better not to print unset parameters at all?